login.html

```diff
@@ -0,0 +1,53 @@
+ <!DOCTYPE html>
+<html>
+<head>
+<link rel = "stylesheet" type = "text/css" href = "login.css">
+</head>
+<body style="position: relative;
+  margin: 0;
+  padding-bottom: 6rem;
+  min-height: 100%;">
+
+	 	<div class= "formBox">
+		<form id="InfoBox" action="Evaluation.html">
+			<h3>
+			Interview Feedback
+			</h3>
+			<div class="loggin">
+				<input type ="text"  placeholder="Username" autocomplete="on" name ="uname" required/>
+				<br>
+				<input type="password" placeholder="Password" autocomplete="off" name="password" required />
+				<br>
+				<button type ="submit" value="Login"> Login</button>
+			</div>
+		</form>
+		</div>
+
+		  <div id="body-wrapper">
+	<div style = "width: 100%" class="info-section" >
+		
+		<article>
+			<h2>Submit new feedback</h2>
+			<img style="width: 33%; display: inline-block;" src="new feedback.jpg" alt="Image not available" >
```

Comment:
Why `33%`? Similarly, why `66%` for the paragraph below? And furthermore, why is it inline?